server/internal/mcpmetadata/getmetadata_test.go

```diff
@@ -0,0 +1,133 @@
+package mcpmetadata_test
+
+import (
+	"testing"
+
+	"github.com/jackc/pgx/v5/pgtype"
+	"github.com/stretchr/testify/require"
+
+	gen "github.com/speakeasy-api/gram/server/gen/mcp_metadata"
+	"github.com/speakeasy-api/gram/server/gen/types"
+	"github.com/speakeasy-api/gram/server/internal/contextvalues"
+	"github.com/speakeasy-api/gram/server/internal/conv"
+	toolsets_repo "github.com/speakeasy-api/gram/server/internal/toolsets/repo"
+)
+
+func TestService_GetMcpMetadata(t *testing.T) {
+	t.Parallel()
+
+	t.Run("fetches metadata with instructions", func(t *testing.T) {
```

Comment:
I kept `t.Run` for consistency with `SetMetadata` tests - but I'd personally prefer having these as separate tests.